.github/.coderabbit.yaml

```
@@ -0,0 +1,129 @@
+    - path: '**/main/java/**/*.java'
+      instructions: |
+        ### [퍼블릭 Java API 변경 보고]
+        - 이 프로젝트는 **안정적·하위 호환 보장**이 필수입니다.
+        - **'Summary by CodeRabbit'** 섹션에,
+          공개 Java API(public·protected 메서드 & `module-info.java`) 변동 사항을
+          **최대 25개**까지 상세 목록화하세요.
+        - 변동 사항이 없으면 *"이번 PR은 공개 Java API 변화가 없습니다."* 라고 명시합니다.
+        - 코드 흐름(비공개 메서드·Java 21 상태 패턴 매칭·Guava 호출 포함)을 **깊이 분석**해
+          - 메서드 추가/삭제
+          - 시그니처 변경(파라미터·반환형)
+          - 동작 변경(사이드이펙트, 예외, 트랜잭션 범위)  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)